library/network/src/lib/y2firewall/firewalld/service.rb

```diff
@@ -86,6 +86,7 @@ def self.modify_ports(name:, tcp_ports: [], udp_ports: [])
       # @param name [String] zone name
       def initialize(name:)
         @name = name
+        @modified = []
```

Comment:
Strictly speaking, that's not needed anymore, right?

Reply:
It was a defensive initialization, just want to avoid exceptions with `<<` or `+=` calls if modified was not called.